watertap/core/membrane_channel_base.py

```diff
@@ -519,17 +551,32 @@ def eq_K(b, t, x, j):
                 == b.properties[t, x].diffus_phase_comp["Liq", j] * b.N_Sh_comp[t, x, j]
             )
 
-        @self.Constraint(
-            self.flowsheet().config.time,
-            self.length_domain,
-            self.config.property_package.solute_set,
-            doc="Sherwood number",
-        )
-        def eq_N_Sh_comp(b, t, x, j):
-            return (
-                b.N_Sh_comp[t, x, j]
-                == 0.46 * (b.N_Re[t, x] * b.N_Sc_comp[t, x, j]) ** 0.36
+        if sherwood_number_eq == SherwoodNumberEq.new:
+
+            @self.Constraint(
```

Comment:
I'm a bit baffled that this doesn't show as covered. Let me try merging in `main`...